index.html

```diff
@@ -342,14 +345,19 @@ <h2>eBraille publication conformance</h2>
 					</ul>
 				</li>
 				<li>SHOULD conform to the accessibility requirements defined in <a href="#ebrl-a11y"></a>.</li>
+				<li>MUST be either an [=unpackaged eBraille file set=] or a [=packaged eBraille file set=] as defined
+					in <a href="#ebrl-fileset"></a>.
+					<div class="note">
+						<p>This is a <a data-cite="infra#willful-violation">willful violation</a> of
+							<a data-cite="epub3/#sec-container-file-and-dir-structure">EPUB publication conformance</a>
+							[[epub3]], which requires a publication to be packaged in an <a 
+							data-cite="epub3#dfn-epub-container">EPUB container</a>.</p>
+					</div></li>
```

Comment:
Another minor nit, but infra says to note the reason for any willful violations so I wonder if we should say that this is to allow publishers to deploy ebraille publications from the web? I'm fine either way, though, as I'm sure we must make this case elsewhere.

Reply:
Right. I didn't add the motivation because it was explained in informative text elsewhere, but it's distant (in the file set intro) so it's better to make it explicit in this note!